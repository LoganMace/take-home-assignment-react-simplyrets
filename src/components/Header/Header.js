import React from 'react';
import './Header.scss';

export const Header = () => {
  return (
    <div className='header'>
      <h1 className='header__title'>Property Listings</h1>
    </div>
  );
}
