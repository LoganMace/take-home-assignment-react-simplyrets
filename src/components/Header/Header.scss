.header {
  z-index: 10;
  height: 103px;
  width: 100%;
  background-color: #F4F4F4;
  display: flex;
  align-items: center;
  position: sticky;
  top: 0;

  &__title {
    font-size: 28px;
    margin-left: 60px;
    font-weight: 600;
  }
}
