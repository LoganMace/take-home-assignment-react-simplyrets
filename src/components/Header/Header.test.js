import React from 'react';
import { render } from '@testing-library/react';
import { Header } from './Header';

describe('Header Component', () => {
  test('should render', () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });

  test('should render header title', () => {
    const { getByText } = render(<Header />);
    const titleText = getByText('Property Listings');
    expect(titleText).toBeInTheDocument();
  });
});
