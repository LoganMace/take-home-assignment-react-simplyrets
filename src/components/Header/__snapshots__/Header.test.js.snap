// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component should render 1`] = `
<DocumentFragment>
  <div
    class="header"
  >
    <h1
      class="header__title"
    >
      Property Listings
    </h1>
  </div>
</DocumentFragment>
`;
